flink/src/main/java/io/delta/flink/source/RowDataBoundedDeltaSourceBuilder.java

```diff
@@ -150,8 +146,8 @@ public RowDataBoundedDeltaSourceBuilder option(String optionName, long optionVal
 
     /**
      * Creates an instance of {@link DeltaSource} for a stream of {@link RowData}. Created source
-     * will work in Bounded mode, meaning it will read content of configured Delta's snapshot
-     * version ignoring all changes done to this table after starting this source.
+     * will work in Bounded mode, meaning it will read the content of the configured Delta Snapshot
```

Comment:
```suggestion
     * will work in Bounded mode, meaning it will read the content of the configured Delta snapshot
```
snapshot is a general concept. It happens to be an internal class name but the user does not need to know that. so the public doc should refer to the concept with lower "s"